x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -73,6 +127,34 @@ describe('Service Overview', () => {
     );
   });
 
+  it('renders transaction latency chart', () => {
```

Comment:
I believe that for these tests that just check charts and tables we don't need to do a full page load before each one. If we just combine them or move them to another describe it seems that the time needed to run goes down from 25-30s to 7-8s.